appinterface/projection/crossfire/crossfire.go

```diff
@@ -182,15 +201,17 @@ func (projection *Crossfire) handleBlockCreatedEvent(
 				"error getting existing validator by block commitment of %s from view %s", signature.ValidatorAddress, err,
 			)
 		}
-		if blockTime.After(projection.phaseOneStartTime) && blockTime.Before(projection.phaseTwoStartTime) {
-			key := constants.ValidatorCommitmentKey(validator.OperatorAddress, constants.PHASE1_COMMIT)
+
+		if blockTime.After(projection.phaseOneStartTime) && blockTime.Before(projection.phaseThreeStartTime) {
+			key := constants.ValidatorCommitmentKey(validator.OperatorAddress, constants.PHASE1N2_COMMIT_PREFIX)
```

Comment:
Explanation: DB key changed to `phase1n2_commit_count` so that count the phase1 and 2 commitment together